torch/distributed/_tensor/README.md

```diff
@@ -47,28 +47,43 @@ Here are some basic DistributedTensor API examples that showcase:
 ```python
 import torch
 import torch.distributed as distributed
-from torch.distributed import DTensor, DeviceMesh, Shard, Replicate, distribute_module
+from torch.distributed._tensor import DTensor, DeviceMesh, Shard, Replicate, distribute_module
 
+world_size = torch.distributed.get_world_size()
 # construct a device mesh with available devices (multi-host or single host)
 device_mesh = DeviceMesh(device_type="cuda", [0, 1, 2, 3])
 # if we want to do row-wise sharding
 rowwise_placement=[Shard(0)]
 # if we want to do col-wise sharding
 colwise_placement=[Shard(1)]
 # distributed tensor returned will be sharded across the dimension specified in placements
-distributed.empty((8, 12), device_mesh=device_mesh, placements=rowwise_placement)
+dist_tensor = DTensor(
```

Comment:
Can we please not change those code examples? The code examples in README are API reference and some of the features not implemented yet, i.e distributed.empty. DTensor(torch.empty()) is not a public API and should not be used here.

Reply:
I see. I have reverted these back!